e2e/LongHaul/device/SystemHealthMonitor.cs

```diff
@@ -20,7 +20,7 @@ internal class SystemHealthMonitor
         private readonly IotHub _iotHub;
         private readonly Logger _logger;
         private static int s_port;
-        private static readonly TimeSpan s_interval = TimeSpan.FromSeconds(3);
+        private static readonly TimeSpan s_interval = TimeSpan.FromSeconds(4);
```

Comment:
we create and send telemetry message every 3 seconds. This caused message backlog with telemetry message delay of 1-2 seconds on average.
To avoid message backlog, we check sending any pending messages every 3 seconds and create telemetry message every 4 seconds.

Reply:
I don't think that is a problem per se. If it was, we can increase the rate at which we check for telemetry to send (reduce from 10 seconds).

Reply:
Note, we may want to change this to even 1/2 a second at some point, so let's fix the other bottleneck.